content/en/contribute/code/workflow.md

```diff
@@ -57,43 +153,6 @@ Timely code reviews are important to getting improvements into the hands of user
 
 Code reviews should be completed within 24 hours of assignment (excluding weekends and holidays). In some cases, a code review may not be possible if a larger discussion needs to be had for design choices or solution objectives, but even in cases like those, some feedback is still to be expected within 24 hours.
```

Comment:
Some of this already mentioned in L112

Reply:
I would keep the two mentions to ensure people are clear on the 24-hour timeliness mark.